portfolio/src/main/webapp/index.html

```diff
@@ -7,30 +7,80 @@
-  <a href="#">Contact</a>
-  <a href="#">Goals</a>
-  </div>
+ 
+ 
 
   <body>  
-    <div id="content">
+      <div id="mySidenav" class="topnav">
+            <a href="javascript:tabPressed('about')">About</a>
+            <a href="javascript:tabPressed('projects')">Projects</a>
+            <a href="javascript:tabPressed('contact')">Contact</a>
+            <a href="javascript:tabPressed('goals')">Goals</a>
+            </div>
+            
+    <div class="row">
+
+     <div id="content">
       <h1>Nitin's Portfolio</h1>
-      <marquee>Latest Update: Learning web app development and practicing machine learning</marquee>
       <hr style="height:1px;border-top:1px solid #0000FF" />
       <img src="images/profile.jpg" alt="Profile Photograph">
+      <div id="about">
       <h3>University of Florida Bachelors in Computer Science</h3>
       <h3>Google STEP Intern</h3>
-      <p>I have been learning how to program for roughly 6 years now. I began with learning web development for an FBLA competition and I now build android applications for national competitions like PennApps and for my personal developer account on the Play Store. Software development is one of my favorite subfields in computer science. Throughout my junior and senior year in high school, I taught students how to program in a computer science club that I founder. My goal is to lead emerging technologies and build software that can make a postive impact in society.</p>
+      <p>
+          I have been learning how to program for roughly 6 years now. I began with learning web development for an FBLA competition and I now build android applications for national competitions like PennApps and for my personal developer account on the Play Store. Software development is one of my favorite subfields in computer science. Throughout my junior and senior year in high school, I taught students how to program in a computer science club that I founder. My goal is to lead emerging technologies and build software that can make a postive impact in society.
+      </p>
+      </div>
+      <div id="projects" style = "display:none";>
+        <h2>Projects:</h2>
+        <ul>
+            <li>Club king Apps mobile app</li>
+            <li>PictoPill application - integrates ArCore and FireBase Vision </li>
+            <li>Tap Media - share social media contacts by tapping phones</li>
+            <li>FBLA Quiz Time app available on the Play Store</li>
+            <li>First app I ever built: Pocket Popular available on Play Store</li>
+        </ul>
+    </div>
+    <div id="contact" style = "display:none";>
+        <h2>Contact </h2>
+        <ul>
+            <li>Instagram: nitin_r_01</li>
+            <li>Snapchat: nitinlikesfood</li>
+            <li>Discord: Nitin Ramadoss</li>
+            <li>Email: nitinramadoss@gmail.com</li>
+        </ul>
+    </div>
+    <div id="goals" style = "display:none";>
+        <h2>Goals </h2>
+        <ul>
+            <li>Lead emergin technologies</li>
```

Comment:
I'll fix this for the next pull request

Reply:
You can fix these within this pull request :). It's usually better to address comments within the same pull request because it's difficult to track previous comments on subsequent ones.